packages/dialect-react/utils/wallets.ts

```diff
@@ -0,0 +1,4 @@
+export enum Wallets {
```

Comment:
Sorry to bring this up again, but can we add more wallets here? We use Phantom, Glow and others. Ideally this should extend the WalletName since it's exactly what we want.

Also does it make sense to update the getWalletName function as well?

And lastly, I think it's conventional for enum names to be singular. To avoid collision we can either name it `SupportedWallet` or (if we don't care, as it was done initially) - `WalletName`. wdyt?